### PR TITLE
Add zip blocking example

### PR DESCRIPTION
**Pull request by oparoz**
An example on how to block zip files

@nickvergessen @jospoortvliet @MorrisJobke

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@oparoz It would be nice to have the regex copy'n'pastable ;)

File: admin_manual/configuration_files/files_access_control.rst [hunk 1]
```diff
@@ -49,6 +49,20 @@ This example blocks access to any folder with the tag ``Confidential``.
     .. figure:: images/files_access_control_collaborative_tags.png
        :alt: Deny access based on collaborative tag
 
+Prevent uploading of specific files
+-------------------------
```
Comment (nickvergessen) [inline admin_manual/configuration_files/files_access_control.rst]: length doesn't match

File: admin_manual/configuration_files/files_access_control.rst [hunk 2]
```diff
@@ -49,6 +49,20 @@ This example blocks access to any folder with the tag ``Confidential``.
...
+
+It's possible to prevent specific files from being uploaded to Nextcloud. You
+simply need to define a rule based on the mimetype and our powerful access control
+engine will block any attempt to upload the file. The safest way to define the rule
+is to use a regular expression, as it will help you cover all the known media types
+used for the type of file you're trying to block.
+
+The following example prevents zip files from being uploaded.
```
Comment (nickvergessen) [inline admin_manual/configuration_files/files_access_control.rst]: You can use
```
``/^application\/(zip|x-zip-compressed)$/i``
```
to insert the example here.

File: admin_manual/configuration_files/files_access_control.rst (hunk 1 above)
Comment (oparoz) [inline admin_manual/configuration_files/files_access_control.rst]: It doesn't matter as it just creates a separator. I think 4 dashes are the minimum.

Comment (nickvergessen) [inline admin_manual/configuration_files/files_access_control.rst]: but building creates a warning:
```
/home/nickv/Nextcloud/Documentation/admin_manual/configuration_files/files_access_control.rst:53: WARNING: Title underline too short.

Prevent uploading of specific files
-------------------------
```

**Comment by oparoz** [comment]
I've added the regexp as text, let me know if this works for you :)

File: admin_manual/configuration_files/files_access_control.rst (hunk 1 above)
Comment (oparoz) [inline admin_manual/configuration_files/files_access_control.rst]: I can't see the page online as the search engine doesn't work...
https://docs.nextcloud.com/server/10/admin_manual/search.html?q=File+access+control#

**Comment by MorrisJobke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
stable10: #163

**Pull request merged**